.env.tpl

```diff
@@ -28,10 +28,10 @@ VPC_SUBNET_C=local-nonsense
 BROKER_STRINGS=local-nonsense
 
 # needed for e2e tests
-CYPRESS_ADMIN_USER_EMAIL=op://mdct_devs/mfp_secrets/CYPRESS_ADMIN_USER_EMAIL
```

Comment:
I assume these were changed in 1password too right?

Reply:
I copied the CYPRESS_ items in 1password and created E2E_ items.  After everything is merged, we can delete the CYPRESS_ versions.